sensing/gnss_poser/include/gnss_poser/gnss_poser_core.hpp

```diff
@@ -91,12 +92,13 @@ class GNSSPoser : public rclcpp::Node
   std::string map_frame_;
 
   sensor_msgs::msg::NavSatFix nav_sat_fix_origin_;
-  bool use_ublox_receiver_;
+  autoware_sensing_msgs::msg::GnssInsOrientationStamped * orientation_msg_;
```

Comment:
I think it is better to use a smart pointer (e.g. shared ptr) than a raw pointer.

Reply:
Or not use pointer. 
like
`autoware_sensing_msgs::msg::GnssInsOrientationStamped orientation_msg_;`

Reply:
Fixed! https://github.com/autowarefoundation/autoware.universe/pull/1230/commits/e7757d4f7d890ba721436139b4cc32ca82f77d81